examples/experimental/i3s-17-and-debug/tile-layer/tile-layer.js

```diff
@@ -83,6 +83,7 @@ export default class TileLayer extends Tile3DLayer {
       }),
       {
         id: `${this.id}-mesh-${tileHeader.id}`,
+        viewportIds: tileHeader.viewportIds,
```

Comment:
Thought: Since the viewportIds is not part of the loaded data on the TileHeader, it could make sense to maintain this in a sub object: `userData`, `renderState` etc.

Reply:
The loaded data is stored in the `tileHeader.header` object. A `tileHeader: TileHeader` instance contain not only loaded data.